### PR TITLE
comment-task-test

### PR DESCRIPTION
**Pull request by chixmxkx**

### CODE REVIEW

**Comment by chixmxkx** [comment]
Wrote feature tests for get comment, add comment and update comment. 
Did centrifugo but something similar already existed in the code base

File: app/Http/Controllers/TaskCommentController.php [hunk 1]
```diff
@@ -34,13 +34,25 @@ public function getCommentsPerTask($taskId)
                 'data' => $result
             ], 200);
         }
-
+        if (isset($result['modified_documents']) && $result['modified_documents'] > 0){
+            // Publish To Centrifugo
+            $this->taskCommentService->publishToCommonRoom(
```
Comment (kayleb01) [inline app/Http/Controllers/TaskCommentController.php:39]: You are supposed to publish the entire comment collection to centrifugo

File: app/Http/Controllers/TaskCommentController.php [hunk 2]
```diff
@@ -34,13 +34,25 @@ public function getCommentsPerTask($taskId)
...
     }
 
+<<<<<<< HEAD
+    public function saveComment(TaskRequest $request, $todoId)
+=======
 
 
     public function saveComment(CommentRequest $request, $todoId)
+>>>>>>> 590e7396f4aaf4f131f4e8d6a7f5bfe98fef54cf
     {
```
Comment (kayleb01) [inline app/Http/Controllers/TaskCommentController.php:56]: Resolve conflicts correctly

**Comment by kayleb01** [comment]
@chixmxkx I have added comments in your code, resolve conflicts properly, publish the entire comment collection and your tst will not pass like that

**Pull request closed without merging**